submissions/xT0PERx/js-dom/code.js

```diff
@@ -0,0 +1,65 @@
+var contentMenu = [
```

Comment:
Use `let`, `const`  declaration instead of `var`

Reply:
Fixed